libs/coroutines/include/hpx/coroutines/stackless_coroutine.hpp

```diff
@@ -121,6 +122,21 @@ namespace hpx { namespace threads { namespace coroutines {
 #endif
         }
 
+#ifdef HPX_HAVE_LIBCDS
+        typedef std::array<std::size_t, 3> cds_type;
+        cds_type get_libcds_data() const
+        {
+            return libcds_data_;
+        }
+
+        cds_type set_libcds_data(cds_type data)
+        {
+            cds_type olddata = libcds_data_;
+            libcds_data_ = data;
+            return olddata;
+        }
+#endif
```

Comment:
Could these functions go into the base class instead (avoids virtual functions and code duplication)?

Reply:
Would you mind telling which file specifically?

Reply:
Please ignore this comment, I was mistaken.